test/Python/src/modal_models/ModalStateReset.lf

```diff
@@ -29,6 +29,10 @@ reactor Modal {
         state counter1(0)
         timer T1(0msec, 250msec)
 
+        reaction(reset) {=
```

Comment:
The rendering of the `reset` trigger seems a bit off:

![image](https://user-images.githubusercontent.com/19938940/173121919-d5c4b2b9-ace4-4295-ba36-688b01801f62.png)

I would actually like to propose an alternative visual syntax. How about something along these lines:
![image](https://user-images.githubusercontent.com/19938940/173123030-6fa9810b-51de-403c-a57e-f133314b2460.png)